scripts/stac/imagery/collection.py

```diff
@@ -105,7 +143,9 @@ def add_capture_area(self, polygons: list[BaseGeometry], target: str, artifact_t
             artifact_target: location where the capture-area.geojson artifact file will be saved.
             This is useful for Argo Workflow in order to expose the file to the user for testing/validation purpose.
         """
-
+        # If published dataset update, merge the existing capture area with the new one
```

Comment:
Do we need to check for a case where the original dataset does not have a capture area but the newly supplied files are processed with one, and not create a capture area in that case?

Reply:
I agree we need to check that